### PR TITLE
Update relayed v3 docs

### PR DESCRIPTION
**Pull request by sstanculeanu**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md [hunk 1]
```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
...
 
-### Creating the inner transactions
+For this, a new command `mxpy tx relay` has been added. The command can be used to relay a previously signed transaction. The saved transaction can be loaded from a file using the `--infile` argument.
 
-We can simply create the inner transactions the same way we did above, by using the `mxpy tx new` command. The only difference is that we'll have to provide an additional argument called `--inner-transactions-outfile`, which represents the file where the inner transactions are saved to be later used by the relayer. To keep it simple, we'll send 1 EGLD from Alice to Bob, and Carol will be the relayer. To create the EGLD transfer transaction from Alice to Bob, we run the following command:
+There are two options when creating the relayed transaction:
+1. Create the inner transaction separately. (Sender signature and relayer signature are added in different steps.)
+2. Create directly the relayed transaction. (Both signatures are added in one step.)
+
+### Creating the inner transaction
```
Comment (popenta) [inline docs/sdk-and-tools/sdk-py/mxpy-cli.md:545]: I am not sure we should still call it `inner transaction`.

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md [hunk 2]
```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
...
+    --gas-limit 100000 --value 1000000000000000000 \
     --proxy https://devnet-gateway.multiversx.com --chain D \
-    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8
-    --inner-transactions-outfile inner_transactions.json
+    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8 \
+    --outfile inner_tx.json
 ```
 
-After creating the inner transaction, we are ready to create the relayed transaction. We have to keep in mind that for `Relayed V3` transactions, the receiver has to be the same as the relayer, in our case, Carol. Another requirement is that the relayed transaction has to have enough gas. The gas is computed by multiplying the base cost (50_000) with the number of inner transactions plus the gasLimit for each inner transaction. For more details on how the gas is computed, check out [this page](/developers/relayed-transactions/#relayed-transactions-version-3).
+After creating the inner transaction, we are ready to create the relayed transaction. We have to keep in mind that for `Relayed V3` transactions.
```
Comment (popenta) [inline docs/sdk-and-tools/sdk-py/mxpy-cli.md:562]: the second sentence can be deleted. (just my tought)

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md [hunk 3]
```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
...
+
+This can be done through `mxpy tx new` command, as follows:
+```sh
+mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/alice.pem --recall-nonce \
+    --receiver erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx \
+    --relayer-pem ~/multiversx-sdk/testwallets/latest/users/carol.pem \
+    --gas-limit 100000 --value 1000000000000000000 \
     --proxy https://devnet-gateway.multiversx.com --chain D \
-    --inner-transactions inner_transactions.json \
+    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8 \
```
Comment (popenta) [inline docs/sdk-and-tools/sdk-py/mxpy-cli.md:584]: maybe `--relayer` and `--relayer-pem` should be next to each other for readability purpose. just my opinion.

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md (hunk 1 above)
Comment (sstanculeanu) [reply docs/sdk-and-tools/sdk-py/mxpy-cli.md:545]: should be ok, for both v1 and v2 inner is used

File: docs/developers/relayed-transactions.md [hunk 4]
```diff
@@ -253,136 +253,65 @@ Decoding the arguments ([useful resources here](/developers/sc-calls-format/)) w
...
-- 1 move balance
+Here's an example of a relayed v3 transaction. Its intent is to call the `add` method of a previously deployed adder contract, with parameter `01`
 
 ```json
 {
   "nonce": 0,
   "value": "0",
-  "receiver": "erd1dcad0dlle658mggthhpvaypjag2et25yk3c2r7p9gjqkgqpxtukqcqakt0",
-  "sender": "erd1dcad0dlle658mggthhpvaypjag2et25yk3c2r7p9gjqkgqpxtukqcqakt0",
+  "receiver": "erd1qqqqqqqqqqqqqpgqeunf87ar9nqeey5ssvpqwe74ehmztx74qtxqs63nmx",
```
Comment (AdoAdoAdo) [inline docs/developers/relayed-transactions.md:307]: data field is missing "YWRkQDAx"

File: docs/developers/relayed-transactions.md [hunk 5]
```diff
@@ -253,136 +253,65 @@ Decoding the arguments ([useful resources here](/developers/sc-calls-format/)) w
...
 :::
 
-Relayed transactions v3 feature comes with a change on the entire transaction structure, adding a new optional field `InnerTransactions`, which is a collection of inner transactions. That being said, relayed transactions v3 allow the user to send multiple inner transactions on the same relayed transaction which will be executed as normal transactions, without the gas consuming data field of the old relayed transactions versions.
+Relayed transactions v3 feature comes with a change on the entire transaction structure, adding two new optional fields: 
+- `relayer`, which is the relayer address that will pay the fees.
+- `relayerSignature`, the signature of the relayer that proves the agreement of the relayer.
 
-In terms of gas limit computation, let's consider the following example: relayed transaction with one inner transaction of type move balance, that also has a data field `test` of length 4.
+That being said, relayed transactions v3 will look and behave very similar to a regular transaction, the only difference being the gas consumption from the relayer. So the consuming data field of the old relayed transactions versions was removed.
```
Comment (AdoAdoAdo) [inline docs/developers/relayed-transactions.md:260]: I think the phrase "So the consuming data field of the old relayed transactions versions was removed." is not very clear.

maybe rephrase to: It is no longer needed to specify the user transaction in the data field.

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md [hunk 6]
```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
...
-We can simply create the inner transactions the same way we did above, by using the `mxpy tx new` command. The only difference is that we'll have to provide an additional argument called `--inner-transactions-outfile`, which represents the file where the inner transactions are saved to be later used by the relayer. To keep it simple, we'll send 1 EGLD from Alice to Bob, and Carol will be the relayer. To create the EGLD transfer transaction from Alice to Bob, we run the following command:
+There are two options when creating the relayed transaction:
+1. Create the inner transaction separately. (Sender signature and relayer signature are added in different steps.)
+2. Create directly the relayed transaction. (Both signatures are added in one step.)
+
+### Creating the inner transaction
+
+The inner transaction is any regular transaction, with the following notes:
+- relayer address must be added
+- extra 50000(base cost) gas must be added for the relayed operation. For more details on how the gas is computed, check out [this page](/developers/relayed-transactions/#relayed-transactions-version-3).
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/mxpy-cli.md:549]: Typo / formatting (parenthesis & spaces).

**Review by andreibancioiu — DISMISSED** [review]

File: docs/sdk-and-tools/sdk-py/mxpy-cli.md [hunk 7]
```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
...
 
-### Creating the inner transactions
+For this, a new command `mxpy tx relay` has been added. The command can be used to relay a previously signed transaction. The saved transaction can be loaded from a file using the `--infile` argument.
 
-We can simply create the inner transactions the same way we did above, by using the `mxpy tx new` command. The only difference is that we'll have to provide an additional argument called `--inner-transactions-outfile`, which represents the file where the inner transactions are saved to be later used by the relayer. To keep it simple, we'll send 1 EGLD from Alice to Bob, and Carol will be the relayer. To create the EGLD transfer transaction from Alice to Bob, we run the following command:
+There are two options when creating the relayed transaction:
+1. Create the inner transaction separately. (Sender signature and relayer signature are added in different steps.)
```
Comment (AdoAdoAdo) [inline docs/sdk-and-tools/sdk-py/mxpy-cli.md:542]: maybe 

1. Create the relayed transaction
2. Add the signatures (sender and relayer)

Comment (AdoAdoAdo) [reply docs/sdk-and-tools/sdk-py/mxpy-cli.md:542]: Now it is only one transaction so the differentiation between an inner transaction and a relayed transaction seems unnecessary

**Review by popenta — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]
